Source/Core/Core/ConfigManager.cpp

```diff
@@ -433,3 +433,38 @@ IniFile SConfig::LoadGameIni(const std::string& id, std::optional<u16> revision)
     game_ini.Load(File::GetUserPath(D_GAMESETTINGS_IDX) + filename, true);
   return game_ini;
 }
+
+std::string SConfig::GetGameTDBImageRegionCode(bool wii, DiscIO::Region region) const
+{
+  switch (region)
+  {
+  case DiscIO::Region::NTSC_J:
+    return "JA";
+  case DiscIO::Region::NTSC_U:
+    return "US";
+  case DiscIO::Region::NTSC_K:
+    return "KO";
+  case DiscIO::Region::PAL:
+  {
+    const auto user_lang = GetCurrentLanguage(wii);
+    switch (user_lang)
+    {
+    case DiscIO::Language::German:
+      return "DE";
+    case DiscIO::Language::French:
+      return "FR";
+    case DiscIO::Language::Spanish:
+      return "ES";
+    case DiscIO::Language::Italian:
+      return "IT";
+    case DiscIO::Language::Dutch:
+      return "NL";
+    case DiscIO::Language::English:
+    default:
+      return "EN";
+    }
+  }
+  case DiscIO::Region::Unknown:
```

Comment:
Looks like you need to turn this into just `return "EN";` to get rid of the build failure.

Reply:
We should keep that enum value and put `return "EN";` outside the `switch` or else macOS will complain not all values are being handled. Or will just replacing with `default` work? I'm not entirely sure how msvc handles switch-cases.

Reply:
Yeah, try `default`.